src/overlays/actors/ovl_Obj_Hgdoor/z_obj_hgdoor.h

```diff
@@ -5,9 +5,18 @@
 
 struct ObjHgdoor;
 
+typedef void (*ObjHgdoorActionFunc)(struct ObjHgdoor*, GlobalContext*);
+
+#define OBJHGDOOR_IS_RIGHT_DOOR(this) \
+    ((this)->dyna.actor.params & 0x8000) // door on the right when looking at the cabinet
```

Comment:
```suggestion
#define OBJHGDOOR_IS_RIGHT_DOOR(thisx) \
    ((thisx)->params & 0x8000) // door on the right when looking at the cabinet
```
At least for the time being, it was decided that params should take a pointer to the base actor instance.